script/repository-list.txt

```diff
@@ -17,7 +17,5 @@ jbpm-wb
 optaplanner-wb
 kie-wb-distributions
 process-migration-service
-optaweb-employee-rostering
-optaweb-vehicle-routing
```

Comment:
this change is not needed since it is replaced by https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/33b1a09e7ccf7ac16add0cdc27c8f6a0bc739ad0/.github/workflows/generate_files.yml#L57 job once PR is merged